### PR TITLE
Misc fixes (TokenSelector + styling)

### LINKED ISSUE #778: react-select selects empty on click outside
1. Open Token selector
2. Enter gibberish
3. Click outside of Token Selector
4. Panic

ezgif-2-ddb9faecf5cd

Happens because react-select changes final selection dynamically (on search, ArrowUp|Down, even hover) not on Enter/Confirmation or menu close

### PR DESCRIPTION
**Pull request by Velenir**
Fixes #778 

Fixes overflow scrollbar on Chrome (not every Chrome even)
localhost_8080_trade_DAI-USDC_sell=0 price=0 99501 from= expires=2880 (1)

Fixes focus change on Enter when inputting gibberish
ezgif-4-04fafca4b93d

Fixes typo in styling in OrderBook

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr959--dexreact.review.gnosisdev.com

File: src/pages/OrderBook.tsx [hunk 1]
```diff
@@ -59,12 +59,12 @@ const OrderBookWrapper = styled.div`
   }
 
   ${InputBox} {
-    padding 0 0 0 3rem;
+    padding: 0 0 0 3rem;
```
Comment (anxolin) [inline src/pages/OrderBook.tsx:62]: shouldn't this have be shown as a compile error or something?

Comment (Velenir) [reply src/pages/OrderBook.tsx:62]: Eslint doesn't catch this or probably anything for css-in-js
Anyone wants to setup [stylelint](https://styled-components.com/docs/tooling#stylelint) ?

Comment (W3stside) [reply src/pages/OrderBook.tsx:62]: would just be an improper css rule and do nothing bad (error wise) because of the semi-colon

**Pull request merged**